cmake/toolchain/linux_arm.cmake

```diff
@@ -7,6 +7,11 @@ set(CMAKE_CXX_COMPILER_ID "GNU")
 set(CMAKE_AR ar CACHE STRING "ar")
 set(CMAKE_CXX_COMPILER_WORKS TRUE)
 set(CMAKE_C_COMPILER_WORKS TRUE)
-set(NO_XCB TRUE)
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mfloat-abi=hard" CACHE STRING "c flags")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mfloat-abi=hard" CACHE STRING "c++ flags")
+set(BUILD_WSI_XCB_SUPPORT OFF CACHE BOOL "" FORCE)
```

Comment:
My CMake-fu is weak, but I thought "FORCE" meant ignore the system variables (see https://github.com/emoon/minifb/issues/54) - is that what you want to do here?  (and in linux_arm64.cmake?)

Reply:
I believe it needs to be FORCE to overwrite the previous CACHE type set command (and the "option" command is equivalent to "set" with CACHED if I understood it correctly). See https://stackoverflow.com/questions/39588836/why-do-i-need-force-to-override-a-cmake-option